### DISCUSSION TITLE
API to stop streaming?

### DISCUSSION THREAD

**Discussion by avi-c** [twitivity/twitter-stream.py#16] [Q&A]
Is there a way for me to stop a stream and later resume it? I'd like to be able to "turn off" the streaming then start it again later.

I've looked at all the docs and the source and can't see a public way to turn off a stream.

**Comment by saadmanrafat** [comment]
I've not seen anything like this either [twittercommunity.com](https://twittercommunity.com) could provide some feedback. What Stream are you using?
